nvmeadm/tests/discovery_test.rs

```diff
@@ -1,7 +1,147 @@
 use nvmeadm::nvmf_discovery::{disconnect, DiscoveryBuilder};
 
+use failure::Error;
+use std::{
+    fs::File,
+    io::prelude::*,
+    net::{SocketAddr, TcpStream},
+    path::Path,
+    process::Command,
+    thread,
+    time::Duration,
+};
+
+static CONFIG_TEXT: &str = "[Malloc]
+  NumberOfLuns 1
+  LunSizeInMB  64
+  BlockSize    4096
+[Nvmf]
+  AcceptorPollRate 10000
+  ConnectionScheduler RoundRobin
+[Transport]
+  Type TCP
+  # reduce memory requirements
+  NumSharedBuffers 32
+[Subsystem1]
+  NQN nqn.2019-05.io.openebs:disk2
+  Listen TCP 127.0.0.1:NVMF_PORT
+  AllowAnyHost Yes
+  SN MAYASTOR0000000001
+  MN NEXUSController1
+  MaxNamespaces 1
+  Namespace Malloc0 1
+# although not used we still have to reduce mem requirements for iSCSI
+[iSCSI]
+  MaxSessions 1
+  MaxConnectionsPerSession 1
+";
+
+static CONFIG_FILE: &str = "/tmp/nvmeadm_nvmf_target.config";
```

Comment:
should these be const?

Reply:
I guess I was expecting the linting and formatting to whine. It did complain about more esoteric stuff (trivial regexps for example).

I'll try const and see.